spec/default_spec.rb

```diff
@@ -48,6 +49,7 @@
     it 'should config JAAS' do
       chef_run.converge(described_recipe)
       expect(chef_run).to create_template('/opt/confluent/confluent-2.0.1/jaas.conf')
+      # rubocop:disable Lint/AmbiguousBlockAssociation
```

Comment:
The preferred style for multi-line blocks is do...end.  Does changing from {} to do...end remove the need for this?